draft-ietf-scitt-software-use-cases.md

```diff
@@ -47,9 +47,8 @@ normative:
 informative:
   RFC9019: SUIT-ARCH
   RFC9124: SUIT-IM
-  TUF:
-    title: >
-      The Update Framework Overview
+  TUF: TUF
```

Comment:
TUF:
    target: https://theupdateframework.io/overview/
    title: The Update Framework Overview